app/mailers/revision_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class RevisionMailer < ActionMailer::Base
+  TO_ADDRESS = 'welsh.editors@moneyadviceservice.org.uk'
+  SUBJECT    = 'Content updated by External Editor'
+
+  default from: ENV['MAILJET_DEFAULT_FROM_ADDRESS']
```

Comment:
If the `fetch` is used and the environment variable is not set, the application will not start due to the fetch error. If the `[]`  is used then I can assume that only the mailer will have the bug. So, we can conclude, that It is a matter of choice on what effect could the worst in case the environment variable is not set.